src/cascade/input_data/db/__init__.py

```diff
@@ -15,3 +7,7 @@ def __getattr__(self, name):
 METRIC_IDS = {"per_capita_rate": 3}
 
 MEASURE_IDS = {"deaths": 1}
+
+db_queries = module_proxy.ModuleProxy("db_queries")
```

Comment:
I would put module_proxy in cascade.core.db and also have these wrapped modules there as well. That way other modules can go `from cascade.core.db import db_queries` without needing to know the gory details.